### ISSUE TITLE
Make Geocoding Provider Selectable

### ISSUE THREAD

**Issue by boldtrn** [graphhopper/graphhopper-navigation-example-legacy#24]
It would be nice if users could select their preferred geocoding provider in the app settings. This would also make testing different providers a lot easier.

**Comment by karussell** [comment]
Yes, but low prio - without autocomplete we should default to provider=nominatim IMO. And once we have pelias then this should be used as default in the US.

**Comment by boldtrn** [comment]
Ok, right now the provider is `provider=default`, see #25. I can change it to nominatim though?

BTW: I think if we make it selectable, it's a nice test app to test the GraphHopper geocoding api. Also making it selectable is not that much work.
